src/rust/lib/graph-editor/src/component/cursor.rs

```diff
@@ -19,27 +19,37 @@ use ensogl::gui::component;
 // === Constants ===
 // =================
 
-const PADDING             : f32 = 2.0;
-const SIDES_PADDING       : f32 = PADDING * 2.0;
-const DEFAULT_SIZE        : V2  = V2(16.0,16.0);
-const DEFAULT_RADIUS      : f32 = 8.0;
-const DEFAULT_COLOR_LAB   : V3  = V3(1.0,0.0,0.0);
-const DEFAULT_COLOR_ALPHA : f32 = 0.2;
+const PADDING        : f32 = 2.0;
+const SIDES_PADDING  : f32 = PADDING * 2.0;
+const DEFAULT_SIZE   : (f32,f32) = (16.0,16.0);
```

Comment:
I assume that `Vector2` is const-incompatible and hence the tuple?

Reply:
Thats right. Changing it to function as discussed via phone